### PR TITLE
Tech Insights plugin is now hosted in backstage community.

### PR DESCRIPTION
**Pull request by hdost**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/tech-insights.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 title: Tech Insights
 author: roadie.io
-authorUrl: https://github.com/RoadieHQ
+authorUrl: https://github.com/backstage-community
```
Comment (freben) [inline microsite/data/plugins/tech-insights.yaml:4]: I think you accidentally got the wrong url here!

Comment (camilaibs) [reply microsite/data/plugins/tech-insights.yaml:4]: Hi @hdost, could you check the comment above so we can merge the pull request 🙂 ?

File: microsite/data/plugins/tech-insights.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 iconUrl: https://roadie.io/images/logos/tech-insights.png
-npmPackageName: '@backstage/plugin-tech-insights'
+npmPackageName: '@backstage-community/plugin-tech-insights'
 tags:
   - tech-insights
   - reporting
   - tech health
   - migrations
-addedDate: '2022-03-31'
+addedDate: '2022-03-31'
```
Comment (freben) [inline microsite/data/plugins/tech-insights.yaml:15]:
```suggestion
addedDate: '2022-03-31'

```

File: microsite/data/plugins/tech-insights.yaml (hunk 1 above)
Comment (freben) [inline microsite/data/plugins/tech-insights.yaml:4]:
```suggestion
authorUrl: https://github.com/backstage/community-plugins/tree/main/workspaces/tech-insights
```

**Comment by freben** [comment]
Oh no. I just realized that this has a DCO problem as well. Gonna close it due to inactivity and make a new one

**Pull request closed without merging**